Examples/Tests/MacroExamples/Implementation/Accessor/EnvironmentValueMacroTests.swift

```diff
@@ -0,0 +1,52 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import MacroExamplesImplementation
+import SwiftSyntaxMacros
+import SwiftSyntaxMacrosTestSupport
+import XCTest
+
+final class EnvironmentValueMacroMacroTests: XCTestCase {
+  private let macros = ["EnvironmentValue": EnvironmentValueMacro.self]
+
+  func testEnvironmentValue() {
+    assertMacroExpansion(
+      """
+      private struct MyEnvironmentKey: EnvironmentKey {
+          static let defaultValue: String = "Default value"
+      }
+
```

Comment:
Since the expansion of the macro is syntactic and doesn’t actually try to resolve `MyEnvironmentKey`, you could remove the declaration of `MyEnvironmentKey`.

Reply:
Fixed: https://github.com/apple/swift-syntax/pull/2565/commits/74a3d68cbc14c55cbe4179ace996f07e279b5238